### ISSUE TITLE
ModException on new game

### ISSUE THREAD

**Issue by TheBrain0110** [DaEgi01/CitiesSkylines-ServiceRadiusAdjuster#1]
I've added a bunch of mods & stuff recently, still testing to see if the game will be stable, so there could be some sort of cross-interaction, dunno. I haven't even tried changing any settings yet, but I got this error popup when loading a new game:

```
The Mod E:\Steam\steamapps\workshop\content\255710\785237088 [YamlDotNet.dll, ServiceRadiusAdjuster.dll] has caused an error [ModException]

Details:
System.FormatException: Input string was not in the correct format
  at System.Int32.Parse (System.String s) [0x00000] in <filename unknown>:0 
  at ServiceRadiusAdjuster.Presenter.OptionItemPresenter.Apply () [0x00000] in <filename unknown>:0 
  at ServiceRadiusAdjuster.Presenter.GlobalOptionsPresenter+<>c.<ApplyAll>b__7_0 (IOptionItemPresenter p) [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[ServiceRadiusAdjuster.Presenter.IOptionItemPresenter].ForEach (System.Action`1 action) [0x00000] in <filename unknown>:0 
  at ServiceRadiusAdjuster.Presenter.GlobalOptionsPresenter.ApplyAll () [0x00000] in <filename unknown>:0 
  at ServiceRadiusAdjuster.Mod.InitializeMod (UIHelperBase helper, System.String modName, System.String modVersion, IConfigurationService configurationService, IGameEngineService gameEngineService) [0x00000] in <filename unknown>:0 
  at ServiceRadiusAdjuster.Mod.OnLevelLoaded (LoadMode mode) [0x00000] in <filename unknown>:0 
  at LoadingWrapper.OnLevelLoaded (UpdateMode mode) [0x00000] in <filename unknown>:0 
```

**Comment by DaEgi01** [comment]
definitely an error of my mod, no cross-interaction involved.
Can you send me your config file, its in 
"%localappdata%\Colossal Order\Cities_Skylines\ServiceRadiusAdjuster_v3.xml"
Either send it via e-mail to daegi01@gmail.com or upload it somewhere and give me a link.

**Comment by DaEgi01** [comment]
I have uploaded an update.
I expect that you won't get the exception anymore... instead you will get an error on the item that is causing the problem. should be easier to find the issue and the mod should work for the other items even in case of an error now.
could you check if it works for you, since I still can't reproduce the error.

**Comment by TheBrain0110** [comment]
It should in theory be a default config, as the error was on first run after installing the mod, and I hadn't touched the configs at all. Unless the problem is that it generated incorrectly...

Gonna go launch the game now and see what it does this time.

[ServiceRadiusAdjuster_v3.zip](https://github.com/DaEgi01/CitiesSkylines-ServiceRadiusAdjuster/files/1419974/ServiceRadiusAdjuster_v3.zip)

**Comment by TheBrain0110** [comment]
So of course, now when I load the game, I don't get any error at all. So either you fixed it, or it was some transient bug in the first place. Although I did see it twice before I submitted it.

Of course, I've also removed a bunch of assets since, so it could've been one of them too. We may never know. 

Instead, I now have an almost-certainly-unrelated issue where I can launch the game, and load a save or start a new map just fine. But then if I quit to the main menu and load a save / start a map again, it crashes during the loading screen with a simulation error. That one's gonna be fun to figure out, but also not usually a big deal in regular play. Anyway, more likely related to the Loading Screen mod, not yours, so I'll stop rambling now :p
